app/views/twoFactorAuth/index.njk

```diff
@@ -0,0 +1,53 @@
+{% extends "../layout.njk" %}
+
+{% block page_title %}
+Sign-in method - GOV.UK Pay
+{% endblock %}
+
+{% block mainContent %}
+<div class="column-full">
+  <a href="{{profile}}" class="link-back">My profile</a>
+</div>
+<div class="column-full">
+  <h1 class="heading-large page-title">
+    Sign-in method
+  </h1>
+</div>
+<div class="column-two-thirds">
+  {% if authenticatorMethod == 'SMS' %}
+    <p>You currently use text message codes to sign in to GOV.UK Pay.</p>
+    <form method="post" action="{{index}}">
+      <input id="csrf" name="csrfToken" type="hidden" value="{{csrf}}"/>
+      <div class="form-group">
+        <button class="button" type="submit">
+          Use an authenticator app instead
+        </button>
+      </div>
+    </form>
+  {% endif %}
+  {% if authenticatorMethod == 'APP' %}
+    <p>You currently use an authenticator app to sign in to GOV.UK Pay.</p>
+    <form method="post" action="{{index}}">
+      <input id="csrf" name="csrfToken" type="hidden" value="{{csrf}}"/>
+      <div class="form-group">
+        <button class="button" type="submit">
+          Set up a different authenticator app
+        </button>
+      </div>
+    </form>
+  {% endif %}
+  <div>
+    <details>
+      <summary>What is an Authenticator app?</summary>
+      <h2 class="heading-small">Authenticator apps</h2>
+      <p>The GOV.UK Pay self-service website requires users to use a security code to sign-in, the code is sent to a users mobile number. It’s possible to get the code from an Authenticator app instead.</p>
```

Comment:
Despite it being in the URL, I don’t think we generally refer to self-service as “self-service” in prose (we call it the “GOV.UK Pay Admin site” or the “service admin site” in the docs if we have to but it’s usually possible to avoid this).

I think this text might work better in the second person. Something like: “Instead of text message codes, you can use an authenticator app to sign in to GOV.UK Pay. This is an app (usually installed on your phone) that generates a security code whenever you need to sign in. If you choose to use an authenticator app, you will only be able to sign in to GOV.UK Pay using that app  and you will no longer receive text message codes.”